test_conformance/select/test_select.cpp

```diff
@@ -368,84 +373,79 @@ static int doTest(cl_command_queue queue, cl_context context, Type stype, Type c
         cmp_stride = block_elements * step * (0xffffffffffffffffULL / 0x100000000ULL + 1);
 
     log_info("Testing...");
-    d = init_genrand( gRandomSeed );
-    uint64_t i;
-    for (i=0; i < blocks; i+=step)
+    d0 = init_genrand( gRandomSeed );
+    randBuf = clCreateBuffer( context, CL_MEM_READ_ONLY, BUFFER_SIZE, nullptr, &err );
+    if( err ) { log_error( "Error: could not allocate src1 buffer\n" );  ++s_test_fail; goto exit; }
+    initSrcBuffer(randBuf, stype, d0);
```

Comment:
It appears that a memobject handle is being treated as a hosptr when passed to initSrcBuffer. This would be incorrect use of the spec.